google/cloud/forseti/common/gcp_api/securitycenter.py

```diff
@@ -138,15 +139,27 @@ def create_finding(self, finding, source_id=None, finding_id=None):
                 '{}/findings/{}'.format(source_id, finding_id),
                 finding
             )
-            LOGGER.debug('Created finding response: %s',
+            LOGGER.debug('Successfully created finding response: %s',
                          response)
             return response
         except (errors.HttpError, HttpLib2Error) as e:
-            LOGGER.exception(
-                'Unable to create CSCC finding: Resource: %s', finding)
-            violation_data = (
-                finding.get('source_properties').get('violation_data'))
-            raise api_errors.ApiExecutionError(violation_data, e)
+            arguments = e.args
+            all_args = []
+            for argument in arguments:
+                all_args.append(argument)
+            raw_error = all_args[1]
+            error = raw_error.decode('utf-8')
+            formatted_error = json.loads(error)
+            error_msg = formatted_error['error']['message']
+            if error_msg == 'Requested entity already exists':
+                LOGGER.debug('CSCC API exception encountered while creating '
```

Comment:
This debug message should be very specific, since you are handling a specific case, and can log it as such.  Something like this:
```
LOGGER.debug('Unable to create finding.  Finding already exists in CSCC. %s', finding)
```

Reply:
Done